packages/dataviews/src/components/dataviews-filters/search-widget.tsx

```diff
@@ -108,85 +144,94 @@ function ListBox( { view, filter, onChangeView }: SearchWidgetProps ) {
 				filter.name
 			) }
 			onFocusVisible={ () => {
+				// `onFocusVisible` needs the `Composite` component to be focusable,
+				// which is implicitly achieved via the `virtualFocus: true` option
+				// in the `useCompositeStore` hook.
+				const compositeStore = compositeStoreRef.current;
+				if ( ! compositeStore ) {
+					return;
+				}
```

Comment:
Since we need to access the store inside a prop for the top-level component, I created a little helper component that reads the store from the `Composite.Context` and passes it back to its parent via a callback.

It is a bit convoluted, but it's kind of a necessity caused by the fact that we're not defining a store outside of `Composite`, nor we're using a `Composite.Provider` that wraps around `Composite`.

It's not pretty, but it seems to do the trick — and hopefully it seems like an edge case. Although I'd be curious to hear what folks think about it, and whether they can come up with good alternatives.